index.d.ts

```diff
@@ -1 +1,2 @@
-export default Map;
+// eslint-disable-next-line @typescript-eslint/ban-types -- It matches Map’s
+export class ManyKeysMap<K, V> extends Map<K[], V> {}
```

Comment:
I think this should be
```suggestion
export default class ManyKeysMap<K, V> extends Map<K[], V> {}
```